### ISSUE TITLE
Statistics

### ISSUE THREAD

**Issue by silentguy256** [SteamGifts/SteamGifts#31]
There is now [a statistics page](http://www.steamgifts.com/stats/personal/community) and a) some of those would look nice in the app (though most of the graphs would probably be hard to port) and b) it would be fun to get notifications if someone adds you to a black or whitelist. ;)

**Comment by mabako** [comment]
While interesting, perhaps not that urgent of an issue :P

Can you look at stats of other users?

**Comment by silentguy256** [comment]
No, afaik only your own stats and global stats... so this was indeed just a low priority "while I'm here anyway" post :)
